Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -1639,8 +1639,7 @@ void TreeResolver::updateForPositionVisibility(RenderStyle& style, const Styleab
     };
 
     // FIXME: Implement via "visibility: force-hidden".
-    if (shouldHideAnchorPositioned())
-        style.setIsForceHidden();
+    style.setIsForceHidden(shouldHideAnchorPositioned());
```

Comment:
Why this change? We should start the style building with the bit set to false. Which test fails without it?

Reply:
Ok, I can check what is going on after landing.